cards.css

```diff
@@ -338,13 +340,13 @@
 .card.modifier.front.left img.cover
 {
     left:                       -25%;
-    clip-path:                  inset(0% 24.5% 0% 24.5%);
+    -webkit-clip-path:                  inset(0% 24.5% 0% 24.5%);
```

Comment:
Indentation here and below, and I think we should keep the `clip-path` declaration in addition to (after) the `-webkit-clip-path` for non-prefixed support.